presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -893,10 +893,12 @@ protected Object visitFunctionCall(FunctionCall node, Object context)
             }
             FunctionHandle functionHandle = metadata.getFunctionManager().resolveFunction(session, node.getName(), fromTypes(argumentTypes));
             FunctionMetadata functionMetadata = metadata.getFunctionManager().getFunctionMetadata(functionHandle);
-            for (int i = 0; i < argumentValues.size(); i++) {
-                Object value = argumentValues.get(i);
-                if (value == null && !functionMetadata.isCalledOnNullInput()) {
-                    return null;
+            if (!functionMetadata.isCalledOnNullInput()) {
```

Comment:
Might need to change `RowExpressionInterpreter::visitCall` as well

Reply:
Code in `RowExpressionInterpreter::visitCall` was structured a little differently so it's less weird. Anyways I added another PR to change that. You can see whether you want to keep that or not.